src/execution_plan/ops/op_apply_multiplexer.c

```diff
@@ -111,22 +66,80 @@ static void _OpApplyMultiplexer_SortChildren(OpBase *op) {
 	}
 }
 
+/* In this init function, the children 1..n are sorted such that branches which are filter ops will be placed at
+ * the begining of the children array, and branches which are apply ops will be placed at the end of the children array,
+ * since filters are easier to evalute. For each branch the init function will collect it argument op, for the injection of
+ * the bounded branch record. */
 OpResult OpApplyMultiplexerInit(OpBase *opBase) {
+	// Sort children.
 	_OpApplyMultiplexer_SortChildren(opBase);
```

Comment:
I thought the branches are already sorted at construction time (`reduce_to_apply`)

Reply:
only the bound branch